### DISCUSSION TITLE
Initial Network Configuration Challenges

### DISCUSSION THREAD

**Discussion by aldebaran02** [Security-Onion-Solutions/securityonion#2667] [Unsupported Versions] [answered]
To anyone in the SO community who has a moment:

I'm setting up a test grid using some hardware here in the house and running into a complication:  

Running SO 2.3.21 from the downloaded ISO (CentOS 7)

Network:
Internet is via 192.168.0.0/24
Defended network is 172.16.0.0/24
Security Onion grid is on 10.0.0.0/24

Grid:
Manager: VM with two NICs - Grid Network (10.0.0.10), Internet-connected (192.168.0.99)
Forward: Bare Metal with two NICs - Grid Network (10.0.0.50), Defended Network (PROMISC)
Storage: Bare Metal with one NIC - Grid Network (10.0.0.20)
TAP in Defended Network

Problem:  I want to collect logs from hosts in the defended network

Stupid Question:  Does this require one of the SO systems to be in that network?  

Less Stupid Question:  If so, which one is the best one (I think Manager was mentioned in documentation)?

Documentation seems to imply that the grid now sits in the defended network and relies on the CentOS firewall to protect it from that network.  It was always my understanding pre-Hybrid Hunter that the SO grid didn't interact with the defended network.

Thanks [again] for your patience!

*Edited because I found the script that was causing my first headache; adding clarifying data about the grid I'm testing.

**Comment by TOoSmOotH** [comment, marked as answer]
There is really only a single network and that is management. The other NIC puts the box in promiscuous mode and it monitors whatever it is plugged into like a span or a TAP. You can put the management interface in any network you prefer and yes the box has its own firewall. As long as you can route other networks to the Manager or the Forward nodes management interface you can send logs to them. Multi-homing the box is not supported right now. If all of your nodes need to be in a separate 10.X space that has no internet then choose the airgap install.

**Reply by aldebaran02** [reply]
Thanks, Mike!  I was hoping to have my cake and eat it, too, but alas physics wins again ;)
